test/test-source-map-generator.js

```diff
@@ -241,42 +241,42 @@ exports['test that source content can be set'] = function (assert) {
   assert.equal(map.sourcesContent[1], null);
 };
 
-exports['test .fromSourceMap'] = function (assert) {
-  var map = SourceMapGenerator.fromSourceMap(new SourceMapConsumer(util.testMap));
+exports['test .fromSourceMap'] = async function (assert) {
+  var map = SourceMapGenerator.fromSourceMap(await new SourceMapConsumer(util.testMap));
```

Comment:
I suppose it's not a big deal to leak the consumers in the tests.

Reply:
Hm, I thought I caught them all, did I miss some?

Reply:
I didn't see where these consumers were destroyed.  But maybe `fromSourceMap` does and I missed it.